File: benchmarks/suite5/bench_runtime.go
package main

import (
	"fmt"
	"os"
)

var suite5BenchSink int64

func suite5PrintI64(n int64) {
	if os.Getenv("SUITE5_BENCH") != "" {
		suite5BenchSink = n
		return
	}
	fmt.Println(n)
}


//go:noinline
func suite5OpaqueI64(x int64) int64 { return x }

File: benchmarks/suite5/bench_runtime.rs
use std::env;

static mut SUITE5_BENCH_SINK: i64 = 0;

pub fn suite5_print_i64(n: i64) {
    if env::var("SUITE5_BENCH").is_ok() {
        unsafe {
            SUITE5_BENCH_SINK = n;
        }
        return;
    }
    println!("{n}");
}
